dbt_platform_helper/domain/copilot_environment.py

```diff
@@ -193,16 +194,16 @@ def generate_cross_account_s3_policies(self, environments: dict, extensions):
                             )
 
         if not resource_blocks:
-            click.echo("\n>>> No cross-environment S3 policies to create.\n")
+            self.io.info("\n>>> No cross-environment S3 policies to create.\n")
             return
 
         for service in sorted(resource_blocks.keys()):
             resources = resource_blocks[service]
-            click.echo(f"\n>>> Creating S3 cross account policies for {service}.\n")
+            self.io.info(f"\n>>> Creating S3 cross account policies for {service}.\n")
             template = self.templates.get_template(S3_CROSS_ACCOUNT_POLICY)
             file_content = template.render({"resources": resources})
             output_dir = Path(".").absolute()
             file_path = f"copilot/{service}/addons/s3-cross-account-policy.yml"
 
             self.file_provider.mkfile(output_dir, file_path, file_content, True)
-            click.echo(f"File {file_path} created")
+            self.io.info(f"File {file_path} created")
```

Comment:
No tests that should be modified now the ClickProvider is being injected?

Reply:
Oh, I thought no need as the behaviours are the same, but perhaps some mocks should have changed?